projection.tex

```diff
@@ -122,73 +126,83 @@ \section{Projection algorithm}
 -\nabla p+\mathbf{D}(\mathbf{v}^{{n}}) \label{e:momentum_pj} 
 \end{align}
 %
-Solve for $\mathbf{v}^{n+1}$, we need to decide $\mathbf{P}$ beforehand. Projection 
-algorithm is implemented with intermediate term $\mathbf{v}^{*}$, the momentum 
-equation is split into 2 parts: 
+Projection method\cite{ferziger_cfd_2019} split eq (\ref{e:momentum_pj}) into
```

Comment:
Conventionally we add citation at the end of a sentence.  But at the time being it is OK to leave it here.  We can come back here to change later.

Reply:
And I wonder if you can pin-point to the original paper for the Projection Method?

Reply:
Sure, we can change it later; yes, this sounds more reasonable to me. Should they be included in a new issue?

Reply:
Sounds like a plan.  Could you file an issue for yourself?